planner/utils/types.py

```diff
@@ -0,0 +1,143 @@
+"""
```

Comment:
Please change the name of this file, to e.g. `poi_types.py`? It currently conflicts with a Python builtin module https://docs.python.org/3/library/types.html.